app/src/main/java/com/example/habitsmasher/FirestoreCallback.java

```diff
@@ -0,0 +1,5 @@
+package com.example.habitsmasher;
+
+public interface FirestoreCallback {
+    void onCallback(Boolean flag);
+}
```

Comment:
This interface implements a `FirestoreCallback`, so we can override the `onCallback` method in `UserRegistrationFragment`.